server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -160,9 +145,16 @@
         return rangeCompare == 0 ? compareFileName(o1.getFile(), o2.getFile()) : rangeCompare;
       });
 
+  // upgrading sequence TsFile resource list
+  private List<TsFileResource> upgradeSeqFileList = new LinkedList<>();
+
   private CopyOnReadLinkedList<TsFileProcessor> closingSequenceTsFileProcessor = new CopyOnReadLinkedList<>();
   // includes sealed and unsealed unSequence TsFiles
   private List<TsFileResource> unSequenceFileList = new ArrayList<>();
+
+  // upgrading sequence TsFile resource list
```

Comment:
maybe the comment is `upgrading unsequence TsFile resource list`

Reply:
fixed